site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -632,6 +632,8 @@ include::docs/v1/tech/shared/_enterprise-edition-blurb.adoc[]
 
 Captcha is supported on multiple theme templates, when enabled. You can further control display on a template by template basis with the link:/docs/v1/tech/themes/template-variables[specific theme template variables].
 
+See link:/docs/v1/tech/themes/helpers#invisible-recaptcha[invisible reCAPTCHA] for information on enabling an invisible captcha badge on the page.
```

Comment:
I don't see this id `invisible-recaptcha` in the `_helpers.ftl` file - did you mean to use `Captcha`?  - you can click test to ensure it goes to the correct section.

Also note, that we should be a little careful about talking about invisible captcha, since reCaptcha v3 and maybe HCaptchaEnterprise support captcha-less CAPTCHA - which we don't particularly support as "invisible" because they don't have a fall back puzzle, and instead the fallback is on the implementor. 

So we should probably specifically say Google reCaptcha v2 using invisible mode or something like that?

Reply:
tbf here Google does call it "invisible reCAPTCHA" on their site. So I'd argue that using that verbiage explicitly is less confusing, and I will try to make it unambiguous where possible https://developers.google.com/recaptcha/docs/invisible